content/en/security/cloud_security_management/setup/without_infrastructure_monitoring.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Setting Up CSM without Infrastructure Monitoring
```

Comment:
I'm leaving this for now because we use a lot of gerunds in security and it's easier 😂 But I'd like to come back and change it at some point because I am not a gerund fan myself!